src/scripts/polyfills/position-sticky-polyfill.coffee

```diff
@@ -0,0 +1,137 @@
+      if !data.clone
+        data.clone = element.clone()
+          .css
+            position: 'fixed'
+            top: data.top
+            bottom: 'auto'
+            left: 'auto'
+          .insertBefore(element)
+        element.css('visibility', 'hidden')
+      else
+        data.clone.css
+          position: 'fixed'
+          top: data.top
+          bottom: 'auto'
+          left: 'auto'
+      onResize(element)
+
+    # Remove Clone
+    else
+      console.log('[Position Sticky Polyfill] Clearing...')
+      if data.clone
+        data.clone.remove()
+        data.clone = null
+        element.css('visibility', 'visible')
+
+  onResize = (element) ->
+    data = element.data('position:sticky')
+    offset = undefined
+    # Make sure no operations that require a repaint are
+    # done unless a cloned element exists
+    if data and data.clone
+      offset = element.offset()
+      data.offsetTop = offset.top
+      data.clone.css
+        left: offset.left
+        width: element.outerWidth()
+
+  doMatched = (rules) ->
+    rules.each (rule) ->
+      elements = ($ rule.getSelectors())
+      declaration = rule.getDeclaration()
+      elements.each (idx, element) ->
+        element = ($ element)
+        data = element.data('position:sticky')
+        if !data
+          data =
+            id: ++uniqueID
+            offsetTop: element.offset().top
+            top: parseInt(declaration.top, 10)
+          element.data('position:sticky', data)
+        onScroll(element)
+        ($ window).on("scroll.position:sticky:#{data.id}", -> onScroll(element))
+        ($ window).on("resize.position:sticky:#{data.id}", -> onResize(element))
+
+  undoUnmatched = (rules) ->
+    rules.each (rule) ->
+      elements = ($ rule.getSelectors())
+      elements.each (idx, element) ->
+        element = ($ element)
+        data = element.data('position:sticky')
+        if data
+          if data.clone
+            data.clone.remove()
+            element.css('visibility', 'visible')
+          ($ window).off(".position:sticky:#{data.id}")
+          element.removeData('position:sticky')
+
+  #
+  # Check to see if the browser supports sticky positioning natively or not. If
+  # it does, we'll just skip the polyfill altogether.
+  #
+  # testElement = document.createElement('div')
+  # try
+  #   # some browsers will throw an error when assigning an
+  #   # unsupported value to a property
+  #   testElement.style.position = 'sticky'
+  # catch e
+  #   return false
+  # # this will return false if the browser doesn't recognize
+  # # "sticky" as a valid position value
+  # return unless testElement.style.position == 'sticky'
+
+  ###*
+  # A unique id used to safely remove event callbacks
+  ###
+  uniqueID = 0
+
+  #
+  # Register the polyfill with Polyfill.js.
+  #
+  Polyfill({ declarations: [ 'position:sticky' ] }, include: [ 'position-sticky' ])
+    .doMatched(doMatched)
+    .undoUnmatched(undoUnmatched)
+
+) jQuery
```

Comment:
No, unfortunately. That would require also observing DOM mutation events which could become heavy and isn't supported out of the box by Polyfill.js. The logical workaround is to make the container sticky instead of the inserted content itself.